src/main.js

```diff
@@ -574,7 +574,7 @@ class MiniGraphCard extends LitElement {
       lines.push(svg`<line x1=${x} y1="0" x2=${x} y2=${height} stroke="${stroke}" stroke-width="0.5"/>`);
     }
 
-    return svg`<g>${lines}</g>`;
+    return svg`<g class="grid-lines">${lines}</g>`;
```

Comment:
Have no idea why - but classes in svg are named with 3 dashes:
![image](https://github.com/user-attachments/assets/378bc56c-19c8-4584-8c67-355d8d6fe263)
so let's blindly change ours to `grid--lines` ))))

Reply:
huh ok why not